common/scala/copyJMXFiles.sh

```diff
@@ -18,6 +18,11 @@
 
 if [[ $( ls /conf/jmxremote.* 2> /dev/null ) ]]
 then
-  mv /conf/jmxremote.* /home/owuser
+  # JMX auth files would be mounted as a symbolic link (read-only mode)
+  # with `root` privileges by the k8s secret.
+  cp -rL /conf/jmxremote.* /home/owuser
+  rm -f /conf/jmxremote.* 2>/dev/null || true
```

Comment:
Note. The files can not be removed in the k8s because it's read-only mounted, so I hide the message and returns true. But in the default (Bare-metal, VM..) environment, the file can be removed.

```
bash-4.4$ rm -f jmxremote.*
rm: can't remove 'jmxremote.access': Resource busy
rm: can't remove 'jmxremote.password': Resource busy
```